.github/workflows/publish-next.yml

```diff
@@ -42,7 +42,7 @@ jobs:
       matrix:
         include:
           - path: ./guides/react/protected-routes
-            tags: '@guides and not (@skip or @todo-guides)'
+            tags: '@guides and not @todo-guides'
```

Comment:
Since `@skip` is supported by default with the latest version of the Cucumber preprocessor, we can remove the handlers for the tag in our Action workflows.